bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -1,57 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" 
```

Comment:
Ok - wouldn’t we be better using the Eclipse inbuilt formatter - at least then can be made consistent if someone else changes it?